ote/src/clj/ote/services/transport.clj

```diff
@@ -556,53 +572,91 @@
                    {::auditlog/name "new-member-email", ::auditlog/value (str (:user_email updated-member))}]
                   ::auditlog/event-timestamp (java.sql.Timestamp. (System/currentTimeMillis))
                   ::auditlog/created-by (get-in requester [:user :id])}]
+    ;; Send email to user to inform about the new membership.
+    (try
+      (email/send!
+        email
+        {:to (:user_email updated-member)
+         :subject title
+         :body [{:type "text/html;charset=utf-8"
+                 :content (str email-template/html-header
+                            (html (email-template/notify-user-new-member updated-member requester operator title)))}]})
+      (catch Exception e
+        (log/warn "Error while sending a email to new member" e)))
+
+    ;; Create auditlog
+    (specql/insert! db ::auditlog/auditlog auditlog)
+
+    ;; Return added new-member data
+    {:id (:user_id new-member)
+     :name (:user_username new-member)
+     :fullname (:user_name new-member)
+     :email (:user_email new-member)}))
+
+(defn invite-new-user [email-config db requester operator user-email]
+  (let [expiration-date (time/sql-date (.plusDays (java.time.LocalDate/now) 1))
+        token (UUID/randomUUID)
+        inserted-token (specql/insert! db ::user/user-token
+                                       {::user/user-email user-email
+                                        ::user/token (str token)
+                                        ::user/operator-id (::t-operator/id operator)
+                                        ::user/expiration expiration-date
+                                        ::user/requester-id (get-in requester [:user :id])})
+        title (str "Sinut on kutsuttu " (:name operator) " -nimisen palveluntuottajan jäseneksi")
+        auditlog {::auditlog/event-type :invite-new-user
+                  ::auditlog/event-attributes
+                  [{::auditlog/name "transport-operator-id", ::auditlog/value (str (::t-operator/id operator))},
+                   {::auditlog/name "transport-operator-name", ::auditlog/value (str (::t-operator/name operator))}
+                   {::auditlog/name "new-member-email", ::auditlog/value (str user-email)}]
+                  ::auditlog/event-timestamp (java.sql.Timestamp. (System/currentTimeMillis))
+                  ::auditlog/created-by (get-in requester [:user :id])}]
+    (try
+      (email/send!
+        email-config
+        {:to user-email
+         :subject title
+         :body [{:type "text/html;charset=utf-8"
+                 :content (html (email-template/new-user-invite requester operator title (::user/token inserted-token)))}]})
```

Comment:
You need to add email-template/html-header
to :content. And if you like you can implement that header with better way than what I came up with.